src/covid_shared/paths.py

```diff
@@ -26,6 +26,7 @@
 DEATHS_OUTPUT_ROOT = Path('/ihme/covid-19/deaths-outputs/')
 INFECTIONATOR_OUTPUTS = Path('/ihme/covid-19/seir-inputs')
 
+MASK_USE_OUTPUT_ROOT = Path('/ihme/covid-19/mask-use-outputs')
```

Comment:
our other covariate output paths don't say "outputs". would it be better to just call it `'/ihme/covid-19/mask-use'`?

Reply:
It would certainly be more consistent. We'd need to check with Austin to see how much work it'll be to update downstream